### DISCUSSION TITLE
Support non-.NET / non-C# Microservice

### DISCUSSION THREAD

**Discussion by HazyFish** [microsoft/aspire#4300] [Ideas]
Aspire seems great for a stack that is entirely in .NET / C#. However, there are always cases where a few microservices are written in other languages/frameworks. I hope there is a way to hook up other non-.NET microservices together in an easy and generic way in Aspire.

**Comment by davidfowl** [comment]
Sure you can use the apphost to wire up nodejs applications or any executable. We have python support here https://github.com/dotnet/aspire/pull/4142.

That said, this is just orchestration, we don't have any immediate plans to build the other pieces for other languages (otel service defaults etc), but they exist and can be wired up https://github.com/dotnet/aspire-samples/tree/main/samples/AspireWithNode.
